lib/container.js

```diff
@@ -283,9 +284,35 @@ function loadGherkinSteps(paths) {
   global.After = fn => event.dispatcher.on(event.test.finished, fn);
   global.Fail = fn => event.dispatcher.on(event.test.failed, fn);
 
-  for (const path of paths) {
-    loadSupportObject(path, `Step Definition from ${path}`);
+  //If gherkin.steps includes only one *.js, then this will iterate through that folder and send all step def js files to loadSupportObject
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment. (spaced-comment)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4548869)